src/core/webidl-index.js

```diff
@@ -42,8 +42,11 @@ export function run(conf, doc, cb) {
   Array.from(document.querySelectorAll("pre.def.idl"))
     .map(elem => {
       const span = document.createElement("span");
-      const clone = elem.cloneNode(true).firstElementChild;
-      span.appendChild(clone);
+      const clone = elem.cloneNode(true).children;
```

Comment:
Nit: `clone` is no longer s good name. Let’s go with:

```js
const [children] = element.cloneNode(true);
```

Reply:
I've updated the code. Please see whether it's good now.